frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-hub.cy.ts

```diff
@@ -52,33 +47,29 @@ describe('Interacting with OperatorHub', () => {
           .invoke('text')
           .should('not.equal', origCatalogTitleTxt);
       });
-  });
 
-  it('filters Operators by name', () => {
+    cy.log('filters Operators by name');
```

Comment:
Could we keep the name filtering and category filtering as separate test cases?
```ts
it('filters Operators by name', () {
  // Test name filtering
}
it('filters Operators by category, () {
  // Test category filtering
}
```

Reply:
We can, but we'll have to refactor the test further since the existing tests have interdependence.

Reply:
Yeah, imo, that would be better than having one monolithic test, but I'll defer to you if it's worth the effort or not.